utils/property-structs.stanza

```diff
@@ -143,9 +143,6 @@ doc:"A well-defined state for a device's power, for example hibernation. \
 public pcb-struct ocdb/property-structs/PowerState <: Equalable&Hashable : 
   name:String,
   powered?:True|False
-defmethod equal? (l:PowerState, r:PowerState) : 
-  name(l)     == name(r) and 
-  powered?(l) == powered?(r)
 defmethod hash (p:PowerState) : 
```

Comment:
You also should remove the hash function

Reply:
I added
```
    with :
      writer => true
      equalable => true
      hashable => true
```
to pcb-structs

Reply:
You also do not need to make pcb-struct inherit from `Equalable&Hashable` explicitly because the macro already makes them inherit from this

Reply:
Done.